src/script/hydra-eval-jobset

```diff
@@ -587,6 +587,7 @@ sub checkJobsetWrapped {
                 updateDeclarativeJobset($db, $project, ".jobsets", $declSpec);
                 $jobset->discard_changes;
                 $inputInfo->{"declInput"} = [ $declInput ];
+                $inputInfo->{"projectName"} = [ fetchInput($plugins, $db, $project, $jobset, "", "string", $project->name, 0) ];
```

Comment:
It really threw me for a loop why this code is so complicated, but this is just making:

```
$inputInfo ->{"projectName"} = [ { "value" => $project->name,  } ]
```

which is used later on by inputsToArgs:

```
    my $alt = $inputInfo->{$input}->[0];

        if ($alt->{type} eq "string") {
            push @res, "--argstr", $input, $alt->{value};
        }
```

which is so complicated because of the support for "alternative" inputs, a largely and perhaps totally dead feature.